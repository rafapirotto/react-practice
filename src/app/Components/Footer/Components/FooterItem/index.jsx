import React from 'react';
import PropTypes from 'prop-types';
import { Link } from 'react-router-dom';

import { DASHBOARD_ROUTE } from '../../../../../app/routes';

const propTypes = {
  title: PropTypes.string.isRequired,
};

const FooterItem = ({ title }) => (
  <div className="pb-2">
    <Link to={DASHBOARD_ROUTE}>{title}</Link>
  </div>
);

FooterItem.propTypes = propTypes;

export default FooterItem;
