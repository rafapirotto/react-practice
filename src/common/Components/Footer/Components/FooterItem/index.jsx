import React from 'react';
import PropTypes from 'prop-types';
import { Link } from 'react-router-dom';

const propTypes = {
  title: PropTypes.string.isRequired,
};

const FooterItem = ({ title }) => (
  <div className="pb-2">
    <Link to="/">{title}</Link>
  </div>
);

FooterItem.propTypes = propTypes;

export default FooterItem;
