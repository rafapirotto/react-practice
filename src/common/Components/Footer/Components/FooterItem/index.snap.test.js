import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';

import FooterItem from '.';

test('It matches the snapshot', () => {
  const component = renderer.create(
    <Router>
      <FooterItem title="example" />
    </Router>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
