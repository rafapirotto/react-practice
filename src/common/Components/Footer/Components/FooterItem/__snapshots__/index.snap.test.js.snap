// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<div
  className="pb-2"
>
  <a
    href="/"
    onClick={[Function]}
  >
    example
  </a>
</div>
`;
